jboss-kie-wildfly-common/configure.sh

```diff
@@ -19,3 +19,29 @@ mkdir -p ${MVN_DIR}/repository
 # Necessary to permit running with a randomised UID
 chown -R jboss:root ${MVN_DIR}
 chmod -R 777 ${MVN_DIR}
+
+# Due to this issue https://github.com/cekit/cekit/issues/593
+# to reduce docker layers in the build phase cleanup is performed
+# here and not in a dedicated module
+
+# Deleting folders domain migration welcome-content
+rm -rf ${JBOSS_HOME}/domain
+rm -rf ${JBOSS_HOME}/migration
+
+# Deleting all *.bat and *.ps1 and init.d folder
+rm -rf ${JBOSS_HOME}/bin/*.bat
+rm -rf ${JBOSS_HOME}/bin/*.ps1
+rm -rf ${JBOSS_HOME}/bin/init.d
+
+# Deleting not needed standalone-*.xml files
+# NOTE: standalone.xml is referenced in the welcome-content
+rm -rf ${JBOSS_HOME}/standalone/configuration/standalone-full.xml
+rm -rf ${JBOSS_HOME}/standalone/configuration/standalone-ha.xml
+rm -rf ${JBOSS_HOME}/standalone/configuration/standalone-full-ha.xml
+rm -rf ${JBOSS_HOME}/standalone/configuration/standalone-load-balancer.xml
+
+# Deleting application properties
+rm -rf ${JBOSS_HOME}/standalone/configuration/application-roles.properties
```

Comment:
there is no need to delete it, the ones that can be removed is mgmt*.properties files